### PR TITLE
update role group description

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #703

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@adefowler - PR has been copy edited and is ready for final review. This is based on issue #703.

Thank you.

File: Office365-Admin/add-users/about-admin-roles.md [hunk 1]
```diff
@@ -109,8 +109,8 @@ You'll probably only need to assign the following roles in your organization.
 |Search editor     |    Can only create, edit, and delete content for Microsoft Search, like bookmarks, Q&A, and locations.     |
 |Service admin     |    Creates service requests for Azure, Microsoft 365, and Office 365 services, and monitors service health.     |
 |Skype for Business admin     | Full access to all Teams and Skype features, Skype user attributes, manages service requests, and monitors service health.      |
-|SharePoint admin     |    Full access to SharePoint Online, manages Office 365 groups, manages service requests, and monitors service health.     |
-|Teams admin     |    Full access to Teams & Skype admin center, manages Office 365 groups and service requests, and monitors service health.     |
+|SharePoint admin     |    Full access to SharePoint Online, manages service requests, and monitors service health.     |
```
Comment (adefowler) [inline Office365-Admin/add-users/about-admin-roles.md]: I'm going to deny this request. Office 365 groups are part of SharePoint admin. If you go to AzureAD and look up the permissions list for SharePoint admin, these are there.

microsoft.directory/groups/unified/basic/update | Update basic properties of Office 365 Groups.
-- | --
microsoft.directory/groups/unified/create | Create Office 365 Groups.
microsoft.directory/groups/unified/delete | Delete Office 365 Groups.
microsoft.directory/groups/unified/members/update | Update membership of Office 365 Groups.
microsoft.directory/groups/unified/owners/update | Update ownership of Office 365 Groups.

File: Office365-Admin/add-users/about-admin-roles.md [hunk 2]
```diff
@@ -109,8 +109,8 @@ You'll probably only need to assign the following roles in your organization.
...
+|Teams admin     |    Full access to Teams & Skype admin center, manages service requests, and monitors service health.     |
```
Comment (adefowler) [inline Office365-Admin/add-users/about-admin-roles.md]: The ability to mange Office 365 groups is an ability of Teams admin. If you go to AzureAD and look up the permissions list for Teams Service admin, these permissions are there:


microsoft.directory/groups/unified/appRoleAssignments/update | Update groups.unified property in Azure Active Directory.
-- | --
microsoft.directory/groups/unified/basic/update | Update basic properties of Office 365 Groups.
microsoft.directory/groups/unified/create | Create Office 365 Groups.
microsoft.directory/groups/unified/delete | Delete Office 365 Groups.
microsoft.directory/groups/unified/members/update | Update membership of Office 365 Groups.
microsoft.directory/groups/unified/owners/update | Update ownership of Office 365 Groups.

**Comment by adefowler** [comment]
Not merging pull request. Both of these roles have permission to manage Office 365 groups.

**Comment by markgort86** [comment]
How can you manage Office 365 groups? As a user with the SharePoint Admin role you can only manage SharePoint sites.

**Pull request closed without merging**